src/main/java/mate/jdbc/services/ManufacturerService.java

```diff
@@ -0,0 +1,18 @@
+package mate.jdbc.services;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerService {
+
+    Manufacturer create(Manufacturer manufacturer);
+
+    Optional<Manufacturer> get(Long id);
```

Comment:
<img width="625" alt="image" src="https://user-images.githubusercontent.com/81781479/211379438-d860d8a2-af88-47ca-ac5c-e5a325419716.png">
what return type should it be?